cmd/flexvolume-driver/driver.go

```diff
@@ -48,7 +48,7 @@ var (
 
 	dockerVolumePluginSocketPath = "/etc/hpe-storage/nimble.sock"
 	stripK8sFromOptions          = true
-	logFilePath                  = "/var/log/hpe-flexvolume-driver.log"
+	logFilePath                  = "/var/log/hpe-docker-driver.log"
```

Comment:
is this right?

Reply:
Thanks for catching, this was meant for common-host-utils